docs/how-to-use-cache.md

```diff
@@ -0,0 +1,88 @@
+
+- Metalake
+- Catalog
+- Schema
+- Table
+- Model
+- Fileset
+- Topic
+- Tag
+- User
+- Group
+- Role
+- ModelVersion
+
+## Configuration Options
+
+| Configuration Key                | Description                                | Default Value          |
+|----------------------------------|--------------------------------------------|------------------------|
+| `gravitino.cache.enabled`        | Whether to enable caching                  | `true`                 |
+| `gravitino.cache.implementation` | Specifies the cache implementation         | `caffeine`             |
+| `gravitino.cache.maxEntries`     | Maximum number of entries allowed in cache | `10000`                |
+| `gravitino.cache.expireTimeInMs` | Cache expiration time (in milliseconds)    | `3600000` (about 1 hr) |
+| `gravitino.cache.enableStats`    | Whether to enable cache statistics logging | `false`                |
+| `gravitino.cache.enableWeigher`  | Whether to enable weight-based eviction    | `true`                 |
+
+- `gravitino.cache.enableWeigher`: When enabled, eviction is based on weight and `maxEntries` will be ignored.
+- `gravitino.cache.expireTimeInMs`: Controls the cache TTL in milliseconds.
+- If `gravitino.cache.enableStats` is enabled, Gravitino will log cache statistics (hit count, miss count, load failures, etc.) every 5 minutes at the Info level.
+
+## Eviction Strategies
+
+Gravitino supports multiple eviction strategies including capacity-based, weight-based, and time-based (TTL) eviction. The following describes how they work with Caffeine:
+
+### Capacity-Based Eviction
+
+When `gravitino.cache.enableWeigher` is **disabled**, Gravitino limits the number of cached entries using `gravitino.cache.maxEntries` and employs Caffeine’s W-TinyLFU eviction policy to remove the least-used entries when the cache is full.
+
+### Weight-Based Eviction
+
+When `gravitino.cache.enableWeigher` is **enabled**, Gravitino uses a combination of `maximumWeight` and a custom weigher to control the total weight of the cache:
+
+- Each entity type has a default weight (e.g., Metalake > Catalog > Schema);
+- Entries are evicted based on the combined weight limit (`maximumWeight`);
+- If a single cache item exceeds the total weight limit, it will not be cached;
+- Weight calculation logic is implemented via `MetadataEntityWeigher`;
+- When this strategy is active, `maxEntries` will be ignored.
+
+### Time-Based Eviction (TTL)
+
+All cache entries are subject to a TTL (Time-To-Live) expiration policy. By default, the TTL is `3600000ms` (1 hour) and can be adjusted via the `gravitino.cache.expireTimeInMs` setting:
+
+- TTL starts at the time of entry creation; once it exceeds the configured duration, the entry expires automatically;
+- TTL can work in conjunction with both capacity and weight-based eviction;
+- Expired entries will also trigger asynchronous cleanup mechanisms for resource release and logging.
```

Comment:
You don't have to have a new doc for cache only, you can move this to `gravitino-server-config.md` as a new chapter. Also, please remember that this is a user-facing doc, so focusing on letting users to understand how to know, no need to introduce the background.